packages/landing/src/url.ts

```diff
@@ -99,6 +102,10 @@ export const WindowUrl = {
             return `${baseTileUrl}/{z}/{x}/{y}.${WindowUrl.ImageFormat}${api}`;
         }
 
+        if (urlType === MapOptionType.VectorTile) {
```

Comment:
if this is `VectorTile` then we should update `Tile` to `RasterTile` 

I prefer Yoda style, `TileVector` `TileRaster`